docs/api.md

```diff
@@ -72,6 +73,7 @@ don't forget to `import asyncio`.
 
 #### ** Python **
 ```python
+from imjoy import api
```

Comment:
Local names and 3rd party names should be imported after standard library names according to PEP8. Separate those three import groups with a blank line. So move this after `import asyncio` and have a blank line between.

Reply:
OK, fixed!